flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -1112,6 +1111,9 @@ private ApplicationReport startAppMaster(
 				case RUNNING:
 					LOG.info("YARN application has been deployed successfully.");
 					break loop;
+				case FINISHED:
+					LOG.info("YARN application has been finished successfully.");
```

Comment:
I'm not sure whether a `warn` is desirable.

Reply:
I think `info` is enough. From Yarn side, the application has finished successfully.